test/helpers/cilium.go

```diff
@@ -17,42 +17,44 @@ package helpers
 import (
 	"bytes"
 	"fmt"
-	"os"
 	"strings"
 	"time"
 
 	"github.com/cilium/cilium/api/v1/models"
 
 	log "github.com/sirupsen/logrus"
+	"os"
```

Comment:
Would be nice to have all stdlib imports in the first block.

Reply:
Good catch! :)

Reply:
Done.